dev-python/black/black-19.10_beta_p0.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+
+inherit distutils-r1
+
+MY_PV="${PV/_beta/b}"
+MY_PV="${MY_PV/_p/}"
+
+DESCRIPTION="The uncompromising Python code formatter"
+HOMEPAGE="https://github.com/psf/black"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${PN}-${MY_PV}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+
+DEPEND="
```

Comment:
Are these actually BDEPENDs?

Reply:
no

Reply:
Maybe I'm misunderstanding?

[16:30:53]  <sam_c> chymera: You've lost me, I'm afraid
[16:31:01]  <sam_c> chymera: Surely all test dependencies are only BDEPENDs?
[16:32:02]  <sam_c> https://devmanual.gentoo.org/general-concepts/dependencies/index.html
[16:32:09]  <sam_c> "BDEPEND specifies dependencies applicable to CBUILD, i.e. programs that need to be executed during the build, "